test/vinyl/ddl.result

```diff
@@ -571,6 +571,7 @@ box.space.test.index.pk
 - unique: true
   parts:
   - type: unsigned
+    sort_order: asc
```

Comment:
This is going to break the ddl integration test, see #8994. @Gumix should be working on the fix. Please talk to him to ensure that his fix will cover your case as well.

Reply:
The change has been dropped from the patchset.

Reply:
Fix is merged with in tarantool/ddl@7ee51b5543bed7f25ac3a314368c238401df4013.